package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "riemann",
-  "version": "0.5.1",
+  "version": "0.6.0",
```

Comment:
Its time for 1.0....hopefully our users are using versioning properly, so they won't be as negatively impacted potentially.